### DISCUSSION TITLE
Issue with Fan Detection in FanControl on Asus x870 motherboard

### DISCUSSION THREAD

**Discussion by Raysurs** [Rem0o/FanControl.Releases#2855] [Q&A]
Hello,

First of all, thank you for this amazing tool. However, I am encountering an issue that I can't resolve.

Problem:
FanControl only detects the two fans of my GPU (Gigabyte Gaming OC RTX 4090). None of the other fans in my system (case and AIO) are showing up in the interface, which prevents me from configuring or controlling them.

My setup:
Case: Lian Li O11D Evo XL
Fans:
- 2 x Noctua NF-A12x25 PWM
- 12 x Noctua NF-A14x25 G2 PWM
Motherboard: ASUS ROG STRIX X870 Gaming-A
Cooling: Arctic Freezer 3 420 (AIO)
CPU: AMD Ryzen 9 9800X3D
GPU: Gigabyte Gaming OC RTX 4090
OS: Windows 11 latest version

Additional details:
All my fans are directly connected to the motherboard using the PMW headers.
They are correctly detected and configurable in the BIOS, using ASUS Q-FAN Control.
I have tried disabling Q-Fan Control in the BIOS, but it didn’t help—my fans still don’t appear in FanControl.
I’m using the latest version of FanControl (v209).

Steps I’ve taken:
Verified all fan connections.
Updated FanControl to the latest version (v209).
Installed the latest drivers for my motherboard.
Disabled Q-Fan Control in the BIOS.
All fans set to PMW in the BIOS.
Tested with other software (e.g., HWMonitor), and fans are detected there.
Installed the FanControl.AsusWMI plugin from 2022, but it doesn't appear in FanControl.

Result:
Only the two fans of my GPU are detected by FanControl. None of the fans connected to the motherboard (case and AIO) are displayed in the application.
![Capture d'écran 2024-11-28 135440](https://github.com/user-attachments/assets/794dd882-3c55-4434-9c8c-fc15ad81582b)
![Capture d'écran 2024-11-28 135556](https://github.com/user-attachments/assets/6b35dce8-20e6-4dc4-b9be-a860872ccaba)

If anyone has any ideas or solutions, I’d greatly appreciate your help. Thank you in advance!

Best regards,

**Comment by Fr0stX76** [comment]
https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/issues/1544

Basically, you will have to wait until the control chip of your MB get supported by LibreHardwareMonitor
